src/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -764,7 +763,7 @@ public bool IsSectionDefined(string name)
         /// In layout pages, renders the content of the section named <paramref name="name"/>.
         /// </summary>
         /// <param name="name">The name of the section to render.</param>
-        /// <returns>Returns <see cref="HtmlString.Empty"/> to allow the <see cref="Write(object)"/> call to
+        /// <returns>An empty <see cref="IHtmlContent"/> to allow the <see cref="Write(object)"/> call to
         /// succeed.</returns>
```

Comment:
@Eilon do we need the `to allow the <see cref="Write(object)"/> call to succeed` bit?